snuba/datasets/entities/sessions.py

```diff
@@ -285,6 +288,7 @@ def get_query_processors(self) -> Sequence[QueryProcessor]:
             TimeSeriesProcessor(
                 {"bucketed_started": "started"}, ("started", "received")
             ),
+            OrganizationRateLimiterProcessor(org_column="org_id"),
```

Comment:
we also want this on outcomes right? This is the only reason I'm requesting changes.

Reply:
I did add this to outcomes. See `outcomes.py` and `outcomes_raw.py`.

Reply:
oh weird, for some reason when I looked at this yesterday I didn't see it. my bad